pkg/proxy/nftables/proxier.go

```diff
@@ -63,29 +63,23 @@ const (
 	// the nftables table
 	kubeProxyTable = "kube-proxy"
 
-	// the services chain in the filter table
-	kubeServicesFilterChain = "KUBE-SERVICES"
+	// service dispatch
+	kubeServicesChain  = "services"
```

Comment:
Should we be prefixing all of these with "kube_" or are they already contextual?  A comment here would help.

Reply:
Everything we do is inside the "kube-proxy" table so the chains/sets don't need any additional prefixing.